dotCMS/src/main/java/com/dotcms/util/user/UserTransformer.java

```diff
@@ -0,0 +1,24 @@
+package com.dotcms.util.user;
+
+import com.dotmarketing.business.DotStateException;
+import com.liferay.portal.model.User;
+import java.util.Map;
+
+
+/**
+ * This interface define the User transformer methods
+ *
+ * @author oswaldogallango
+ * @since 2017
+ */
+public interface UserTransformer {
+
+    /**
+     * Return a com.liferay.portal.model.User from the Map<String, Object> object
+     * @param map DB map returned by the dotconnect user seach
+     * @return User
+     * @throws DotStateException
+     */
+    User fromMap(Map<String, Object> map) throws DotStateException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)